.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -43,22 +43,7 @@ body:
         | This proposal will allow end users to accomplish X | Should |
         | This proposal will allow developers to accomplish Y | Could |
         | This proposal will allow end users to accomplish Z | Won't |
-  - type: input
-    id: files-version
-    attributes:
-      label: Files Version
-      description: Which version of Files are you using?
-      placeholder: v1.5
-    validations:
-      required: true
-  - type: input
-    attributes:
-      label: Windows Version
-      description: Which version of Windows are you using? Press <kbd>Win</kbd> + <kbd>Pause/Break</kbd> and copy.
-      placeholder: e.g. Windows 11 21H2 22000.194
-    validations:
-      required: true
```

Comment:
Can you keep this part of the template? The version number is still useful to have for feature requests.

Reply:
We should at least make it optional, since it's not relevant to every PR.